DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests/SRSnapshotTests.swift

```diff
@@ -21,257 +21,186 @@ final class SRSnapshotTests: SnapshotTestCase {
     func testBasicTexts() throws {
         show(fixture: .basicTexts)
 
-        var image = try takeSnapshot(configuration: .init(privacy: .allowAll))
-        DDAssertSnapshotTest(
-            newImage: image,
-            snapshotLocation: .folder(named: snapshotsFolderName, fileNameSuffix: "-allowAll-privacy"),
-            record: recordingMode
-        )
-
-        image = try takeSnapshot(configuration: .init(privacy: .maskAll))
-        DDAssertSnapshotTest(
-            newImage: image,
-            snapshotLocation: .folder(named: snapshotsFolderName, fileNameSuffix: "-maskAll-privacy"),
-            record: recordingMode
-        )
+        try forEachPrivacyMode { privacyMode in
+            let image = try takeSnapshot(configuration: .init(privacy: privacyMode))
+            DDAssertSnapshotTest(
+                newImage: image,
+                snapshotLocation: .folder(named: snapshotsFolderName, fileNameSuffix: "-\(privacyMode)-privacy"),
+                record: recordingMode
+            )
+        }
```

Comment:
Added `forEachPrivacyMode { _ in }` helper to not repeat the same dance for new mode.